yaksh/views.py

```diff
@@ -2245,17 +2246,26 @@ def download_sample_csv(request):
 
 @login_required
 @email_verified
-def duplicate_course(request, course_id):
+def duplicate_course(request, copy_type, course_id):
     user = request.user
     course = Course.objects.get(id=course_id)
     if not is_moderator(user):
         raise Http404('You are not allowed to view this page!')
 
     if course.is_teacher(user) or course.is_creator(user):
-        course.create_duplicate_course(user)
+        if copy_type == "copy":
```

Comment:
should this be deep and shallow? ` copy` keyword as a copy_type is pretty ambiguous.